spec/support/user_header_spec_helper.rb

```diff
@@ -0,0 +1,73 @@
+module UserHeaderSpecHelper
```

Comment:
we can dump this file if we just go with a simple hash {'a' => 1, 'b'=>2} for headers

Reply:
Same